client/src/components/campaign-old/campaign.scss

```diff
@@ -1,6 +1,6 @@
 @import '../../styles/variables';
 
-mont-campaign {
```

Comment:
what's the plan for all the `*-old` stuff? will we be removing it soon?

Reply:
Yes.